custom-prom.md

```diff
@@ -8,6 +8,10 @@ Custom Prometheus and Grafana
 
 When integrating Kubecost with an existing Prometheus, we recommend first installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)) as a dry run before integrating with an external Prometheus deployment. You can get in touch (support@kubecost.com) or via our [Slack community](https://join.slack.com/t/kubecost/shared_invite/enQtNTA2MjQ1NDUyODE5LWFjYzIzNWE4MDkzMmUyZGU4NjkwMzMyMjIyM2E0NGNmYjExZjBiNjk1YzY5ZDI0ZTNhZDg4NjlkMGRkYzFlZTU) for assistance.
 
+The Kubecost Prometheus deployment is used both as a source and a sink for cost & capacity metrics. It’s optimized to not interfere with other observability instrumentation and by default only contains metrics that are useful to the Kubecost product. This amounts to retaining __70-90% fewer metrics__ than a standard Prometheus deployment.
+
+For the best experience, we generally recommend teams use the bundled prometheus-server & grafana but reuse their existing kube-state-metrics and node-exporter deployments if they already exist. This setup allows for the easiest installation process, easiest ongoing maintenance, minimal duplication of metrics, and more flexible metric retention.
```

Comment:
> kube-state-metrics 

This is actually now optional as I understand it. @AjayTripathy can you help clarify. Might be able to remove below as well.

Reply:
KSM is now optional, that's correct. That being said, we don't heavily test this usecase and having KSM gives us some redundancy for accurate pod times (if kc is down, ksm is up and vice versa). I'd leave it in for now until HA mode at which point we could potentially truly deprecate KSM.